doc/sphinx/crystal_plasticity.rst

```diff
@@ -205,6 +205,22 @@ orientation update equations must be supplemented with the evolution
 equations for the internal variables, :math:`\dot{\mathbf{h}}`, which are
 integrated implicitly.
 
+In addition to this basic kinematic framework, NEML has the ability
+to provide crystal models with the Nye tensor
+
+.. math::
+   \bm{\alpha} = -\nabla \times \mathbf{F}^{e-1}
+
+Describing the areal density of geometrically necessary dislocations.
+The Nye tensor can be used as an *ad hoc* contributor to slip system
+flow or hardening models.  The NEML system itself cannot calculate
+the Nye tensor itself -- that requires gradient information not available
+at the material point level -- but can accept the calculated Nye tensor
+from a higher level framework, for example an FEA model, and process it
+accordingly.
+To help with this calculation, the model provides the inverse elastic
+tensor :math:`\mathbf{F}^{e-1}` as an available output.
```

Comment:
We need to mention in which configuration is the curl taken, I believe in this definition it is in the current configuration, for reference one can look into Das, Hofmann and Tarleton 2018.

Reply:
Yes, it's the current configuration.  I'll note it in the docs.